src/components/table/PaginatedTable.js

```diff
@@ -206,7 +208,7 @@ function PaginatedTable({
                                 />
                             </TableRow>
                         )}
-                    </TableFooter>
+                    </TableBody>
```

Comment:
Does this not use the `TableFooter` any more?

Reply:
I couldn't see what TableFooter was providing... Do you have any ideas?

Reply:
I assumed it was formatting for the paginated arrows and stuff, but tbh I haven't ever properly looked at this code!

Reply:
lucky you....